README.md

```diff
@@ -670,7 +670,7 @@ Static methods and type properties work similarly to global functions and global
 
 Declare variables and function return types as optional with `?` where a nil value is acceptable.
 
-Use implicitly unwrapped types declared with `!` only for instance variables that you know will be initialized later before use, such as subviews that will be set up in `viewDidLoad`.
+Use implicitly unwrapped types declared with `!` only for instance variables that you know will be initialized later before use.
```

Comment:
Actually, I would rewrite this sentence to don't use `!` at all. According to swiflint rules force unwrap is forbidden 🤔 What do you think about it?

Reply:
It's a good idea to emphasise that usage of `!` is discouraged, but I'd not forbid it, since it might be useful (well, maybe not useful, but more convenient for sure) when working with legacy, old-style frameworks, for example.

Reply:
I'm partly agree with you, but it isn't hard to do `guard let` or `if let` that will make code more swifty and safety even if it is old-style (objc) frameworks